LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Human Cell Atlas Authors, https://humancellatlas.org
+Copyright (c) 2019 Human Cell Atlas Authors, https://humancellatlas.org
```

Comment:
```suggestion
Copyright (c) 2017-2019 Human Cell Atlas Authors, https://humancellatlas.org
```
Based on Rex and Marcus's comments, the suggested change appears to be a correct approach.